hawkular-alerts-rest/src/main/java/org/hawkular/alerts/rest/TriggersHandler.java

```diff
@@ -514,45 +839,84 @@ public Response createCondition(
             final String triggerId,
             @ApiParam(value = "Json representation of a condition. For examples of Condition types, See "
                     + "https://github.com/hawkular/hawkular-alerts/blob/master/hawkular-alerts-rest-tests/"
-                    + "src/test/groovy/org/hawkular/alerts/rest/ConditionsITest.groovy")
+                    + "src/test/groovy/org/hawkular/alerts/rest/ConditionsITest.groovy")//
             String jsonCondition) {
```

Comment:
At least for now I leveraged JacksonDeserializer.deserializeCondition to re-use code and eliminate the big code block below.